app/main/views/audits.py

```diff
@@ -83,23 +90,43 @@ def list_audits():
     if object_type:
         if object_type not in AUDIT_OBJECT_TYPES:
             abort(400, 'invalid object-type supplied')
-        if not object_id:
-            abort(400, 'object-type cannot be provided without object-id')
```

Comment:
It was specifically an index-related thing. This is where I would have to actually check the use of this index but will need to play with my db to do so.